tests/builder/test_base.py

```diff
@@ -202,7 +202,8 @@ def test_replace_macros(macro_spec_path, prepared_macros):
     result_macros = dict(MACRO_REGEX.findall(spec_contents))
 
     # All expected macros have their contents replaced
-    assert all(val == 'REPLACED' for val in result_macros.values()), result_macros
```

Comment:
Perhaps, if you do not know how to squash and reabase for your commits?

This is the way.

```
git rebase -i a_commit_ref_before_your_1st_commit
```